_posts/2022-07-26-bea17-ambassador-paper.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Ambassador Paper Award at the 17th BEA Workshop
+author: bea
+author_profile: true
+excerpt: '"Educational Multi-Question Generation for Reading Comprehension" by Manav Rathod, Tony Tu, and Katherine Stasaski'
+tags:
+    IAALDE
+    ambassador
+categories:
+    blog
+toc: false
+published: true
+---
+
+As we are part of the International Alliance to Advance Learning in the Digital Era ([IAALDE](https://alliancelss.com/)), every year we select an ambassador paper that will represent our society (SIGEDU) at another conference affiliated with a "sister" society from IAALDE. All BEA workshop attendees can vote for their favorite ambassador paper (either an oral presentation or a poster).
+
+The following paper was voted the ambassador paper at the [17th BEA workshop](https://sig-edu.org/bea/2022): 
+
+> **_Educational Multi-Question Generation for Reading Comprehension_** <br>
+> by Manav Rathod, Tony Tu and Katherine Stasaski
+
+The authors will be offered an opportunity to present their paper as a talk at another IAALDE conference in 2023.
+
+The honorable mentions are:
+
+#2. *The Specificity and Helpfulness of Peer-to-Peer Feedback in Higher Education* (Roman Rietsche, Andrew Caines, Cornelius Schramm, Dominik Pfütze and Paula Buttery)
+
+#3. *On Assessing and Developing Spoken ’Grammatical Error Correction’ Systems* (Yiting Lu, Stefano Bannò and Mark Gales)
```

Comment:
Are some of these numbers supposed to be the same? Are they ties in ranks?

Reply:
Yes!